dev-requirements.txt

```diff
@@ -0,0 +1,36 @@
+#
```

Comment:
I can move the dependency versioning and related workflow modifications to a separate PR if you think that's more convenient.